include/spatialindex/capi/sidx_config.h

```diff
@@ -42,10 +42,8 @@
   typedef unsigned __int16 uint16_t;
   typedef unsigned __int32 uint32_t;
   typedef unsigned __int64 uint64_t;
-#endif
-
-#if _MSC_VER >= 1900
-   #include <cinttypes>
+#else
+  #include <stdint.h>
```

Comment:
use `cstdint` instead.

Reply:
cstdint is a C++ header. Why do you want to include a C++ header in a C API?